src/app/heroes/heroes.component.css

```diff
@@ -0,0 +1,51 @@
+.heroes {
```

Comment:
Consider switching to scss to get a slightly cleaner style files.

Reply:
The tutorial I'm using uses CSS, I will add switching to SCSS as a stretch goal once finished.